values/decoding/varint_int_iterator.go

```diff
@@ -3,19 +3,21 @@ package decoding
 import (
 	"encoding/binary"
 	"io"
+
+	xio "github.com/xichen2020/eventdb/x/io"
 )
 
 // varintIntIterator iterates over a stream of
 // varint encoded int data.
 type varintIntIterator struct {
-	reader io.ByteReader
+	reader xio.SimpleReadCloser
```

Comment:
Store a separate byteReader to save the interface conversion cost at line 31. See values/decoding/raw_size_string_iterator.go

Reply:
Same comment as above.